jaeger-thrift/src/main/java/io/jaegertracing/thrift/internal/senders/HttpSender.java

```diff
@@ -119,13 +134,68 @@ public Builder withAuth(String authToken) {
       return this;
     }
 
+    public Builder withCertificatePinning(String[] sha256certs /* comma separated */) {
```

Comment:
What is comma separated that the comment refers to?

Reply:
That _was_ comma separated but the argument itself is already processed to an array so it doesn't deal with the format. Just deleted, thank you to notice me.